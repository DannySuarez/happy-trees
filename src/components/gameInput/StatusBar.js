import React from 'react';
import PropTypes from 'prop-types';
import styles from './statusbar.css';


function StatusBar({ nickname, handleChange }) {
  return (
    <div className={styles.StatusBar}>
      <div className={styles.round}>1 of 5</div>
      <div className={styles.timerContainer}>timer</div>
      <div className={styles.currentArtist}>Drawer: {nickname}</div>
      <div className={styles.colorPicker}>
        {/* <label className={styles.brushTitle}htmlFor="brush">Brush Color</label>
        <input className={styles.brushBar} type="color" id="brush" name="brush" value="#FF914D" onChange={handleChange}/> */}
      </div>
    </div>
  );

}
StatusBar.propTypes = {
  nickname: PropTypes.string,
  handleChange: PropTypes.func
};

export default React.memo(StatusBar);
