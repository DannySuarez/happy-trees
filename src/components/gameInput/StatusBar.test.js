import React from 'react';
import { shallow } from 'enzyme';
import StatusBar from './StatusBar';

describe('StatusBar', () => {
  it('renders Status Bar component', () => {
    const wrapper = shallow(<StatusBar />);
    expect(wrapper).toMatchSnapshot();
  });
});
