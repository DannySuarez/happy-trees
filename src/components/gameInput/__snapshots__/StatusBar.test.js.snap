// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StatusBar renders Status Bar component 1`] = `
<div
  className="StatusBar"
>
  <div
    className="round"
  >
    1 of 5
  </div>
  <div
    className="timerContainer"
  >
    timer
  </div>
  <div
    className="currentArtist"
  >
    Drawer: Danny
  </div>
  <div
    className="colorPicker"
  />
</div>
`;
